### ISSUE TITLE
Report error "failed to insert profile: timestamp must not be zero" on v0.11.1

### ISSUE THREAD

**Issue by mangox** [parca-dev/parca#1072]
Error:
level=error name=parca ts=2022-05-24T07:26:54.961974115Z caller=scrape.go:500 ... msg="WriteRaw failed for scraped profile" err="rpc error: code = Internal desc = **failed to insert profile: timestamp must not be zero**"

profiling config in parca.yml.
profiling_config:
      pprof_config:
        fgprof:
          enabled: true
          path: /debug/fgprof
          delta: true

**Comment by knrt10** [comment]
I think I found the root cause. [This line](https://github.com/parca-dev/parca/blob/main/pkg/scrape/target.go#L373) is adding seconds but subtracting 1 from it, due to which scraper is not picking it up correctly. As mentioned in [parca.yaml](https://github.com/parca-dev/parca/blob/main/parca.yaml#L20) it should be in `?seconds=<scrape-interval>` format but instead it is in `?seconds=<scrape-interval> - 1`.

If this is correct, I can open a PR for it. WYD cc @brancz?

**Comment by brancz** [comment]
Yeah unfortunately we realized that fgprof doesn't set several fields that we require to be set, I already opened several PRs to fix it: https://github.com/felixge/fgprof/pulls?q=is%3Apr+author%3Abrancz+

**Comment by mangox** [comment]
@brancz Cool. Looks those PRs could solve my issue. Hope they could be merged ASAP. :)
I'll try to build it locally using your commits.
